pkg/rpc/response/result/version.go

```diff
@@ -1,16 +1,37 @@
 package result
 
-import "github.com/nspcc-dev/neo-go/pkg/config/netmode"
+import (
+	"github.com/nspcc-dev/neo-go/pkg/config/netmode"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/fixedn"
+)
 
 type (
 	// Version model used for reporting server version
 	// info.
 	Version struct {
+		// Magic contains network magic.
+		// Deprecated: use Protocol.StateRootInHeader instead
 		Magic     netmode.Magic `json:"network"`
 		TCPPort   uint16        `json:"tcpport"`
 		WSPort    uint16        `json:"wsport,omitempty"`
 		Nonce     uint32        `json:"nonce"`
 		UserAgent string        `json:"useragent"`
+		Protocol  Protocol      `json:"protocol"`
+		// StateRootInHeader is true if state root is contained in block header.
+		// Deprecated: use Protocol.StateRootInHeader instead
+		StateRootInHeader bool `json:"staterootinheader,omitempty"`
+	}
+
+	// Protocol represents network-dependent parameters.
+	Protocol struct {
+		AddressVersion              byte          `json:"addressversion"`
+		Network                     netmode.Magic `json:"network"`
+		MillisecondsPerBlock        int           `json:"msperblock"`
+		MaxTraceableBlocks          uint32        `json:"maxtraceableblock"`
```

Comment:
`json:"maxtraceableblocks"`

Reply:
@fyrchik?

Reply:
Fixed.

Reply:
@fyrchik, you might've forgotten to update PR, not fixed yet.

Reply:
accidentally changed `maxtransactionperblock`, fixed now.